pkg/doc/signature/suite/jsonwebsignature2020/public_key_verifier.go

```diff
@@ -11,24 +11,45 @@ import (
 	"crypto/ecdsa"
 	"crypto/elliptic"
 	"errors"
+	"fmt"
 	"math/big"
 
+	"github.com/decred/dcrd/dcrec/secp256k1/v2"
```

Comment:
have you tried  github.com/btcsuite ? I believe they have secp256k1 alg as well.

Reply:
I didn't. `github.com/decred` was initially added by @sudeshrshetty  but I think we can switch to `github.com/btcsuite`.

Reply:
@kdimak @Baha-sk Switching that import is not part of this PR. Consider followup if there is a reason to do so.